a-to-z.md

```diff
@@ -27,72 +32,94 @@ Use the following terms to refer to specific types of app:
 * R Shiny app
 * static web app
 
-### Concourse
+## Concourse
+
 Upper case.
 
-### conda
+## conda
```

Comment:
As we have conda in here, it might be worth adding renv/Renv and the convention we will use for that.